UI/data/locale/en-US.ini

```diff
@@ -801,6 +801,7 @@ Basic.Settings.General.ScreenSnapping="Snap Sources to edge of screen"
 Basic.Settings.General.CenterSnapping="Snap Sources to horizontal and vertical center"
 Basic.Settings.General.SourceSnapping="Snap Sources to other sources"
 Basic.Settings.General.SnapDistance="Snap Sensitivity"
+Basic.Settings.General.SpacingHelpers="Show pixel spacing helper lines"
```

Comment:
Users might expect the word "alignment" and/or "guides" in here somewhere.

Reply:
Changed to "Show pixel alignment guides"